config/routes/api/v2.rb

```diff
@@ -223,6 +223,7 @@ class ActionDispatch::Routing::Mapper
             get :releases
             put :refresh_subscriptions
             put :content_override
+            get :content_override, :action => :product_content
```

Comment:
Okay - I wasn't a huge fan of how I enacted these methods in activation keys and was trying to match them while making it better. However, I'll change the route to get :product_content to keep it comparable to how it is already enabled in activation keys